apps/fluent-tester/src/RNTester/FabricTester.tsx

```diff
@@ -25,6 +25,86 @@ export interface IFabricTesterProps {
   enabledTests: TestDescription[];
 }
 
+export let app = 'office';
+
+const Header: React.FunctionComponent<{}> = () => {
+
+  const [selectedPlatform, setSelectedPlatform] = React.useState('win32');
+  const [selectedApp, setSelectedApp] = React.useState('office');
+  const [selectedTheme, setSelectedTheme] = React.useState('default');
+
+  const themeColor = () => {
+
+    let color = 'black'; // default: office (black)
+
+    if (selectedApp == 'office') color = 'black';
+    if (selectedApp == 'word') color = '#2B579A';
+    if (selectedApp == 'excel') color = '#217346';
+    if (selectedApp == 'powerpoint') color = '#B7472A';
+    if (selectedApp == 'outlook') color = '#106EBE';
+
+    return { color: color };
+  }
+
+  const onAppChange = React.useCallback((appValue: string) => {
+    app = appValue;
+    setSelectedApp(appValue);
+  }, []);
+
+  return (
+    <View style={fabricTesterStyles.header}>
+      <Text style={[fabricTesterStyles.testHeader, themeColor()]} testID={BASE_TESTPAGE}>
+        ⚛ FluentUI Tests
+      </Text>
+
+      <View style={fabricTesterStyles.pickerRoot}>
+        <View style={fabricTesterStyles.picker}>
+          <Text style={fabricTesterStyles.pickerLabel}>Platform:  </Text>
+          <Picker
```

Comment:
Platform emulated themes would be another one of those things to add to the TODO/Next steps issue :)